net/net.go

```diff
@@ -64,6 +64,11 @@ type NetInterfaceStat struct {
 	Addrs        []NetInterfaceAddr `json:"addrs"`
 }
 
+type NetFilterStat struct {
+	ConnTrackCount	int64	`json:"conntrackcount"`
+	ConnTrackMax	int64	`json:"conntrackmax"`
```

Comment:
Could you change these json name to snake_case, such as `conn_track_max` ?
(I know JSON use camelCase on their name, but I had not know that at first, and other APIs already have snake_case.)